### PR TITLE
Fixes #153 updateLastAccessed to handle arrays of tabs

### LINKED ISSUE #153: `checkToClose` throwing exception when calling `updateLastAccessed`
Commit d305eb811a13954ee7bc7a78a3e4fb53c082458c introduced a bug. [`chrome.tabs.getSelected`](https://developer.chrome.com/extensions/tabs#method-getSelected) returns a `Tab`, but [`chrome.tabs.query`](https://developer.chrome.com/extensions/tabs#method-query) returns an `Array<Tab>`. v5.3.1 is throwing exceptions when calling `checkToClose`. Since `chrome.tabs.query` is the latest API and always deals with an `Array<Tab>`, `tabmanager.updateLastAccessed` should likely deal with an `Array<Tab>` as well.

https://github.com/tabwrangler/tabwrangler/commit/d305eb811a13954ee7bc7a78a3e4fb53c082458c#commitcomment-24176015

### PR DESCRIPTION
**Pull request by AsaAyers**

### CODE REVIEW

File: app/js/tabmanager.js [hunk 1]
```diff
@@ -225,9 +227,11 @@ const TabManager = {
    * @param {mixed} tabId
    *  Tab ID or Tab object.
    */
-  updateLastAccessed(tabOrTabId: chrome$Tab | number) {
+  updateLastAccessed(tabOrTabId: TabOrTabId | Array<TabOrTabId>) {
```
Comment (ssorallen) [inline app/js/tabmanager.js]: I believe this should be the following:

    chrome$Tab | number | Array<chrome$Tab>

[`chrome.tabs.query`](https://developer.chrome.com/extensions/tabs#method-query) always passes an array of tabs to the `callback` param; the array won't ever contain IDs.

Comment (AsaAyers) [reply app/js/tabmanager.js]: While there is nothing at the moment that sends an array of IDs, it is supported by that code.

Comment (ssorallen) [reply app/js/tabmanager.js]: Okay sure.

File: app/js/tabmanager.js [hunk 2]
```diff
@@ -225,9 +227,11 @@ const TabManager = {
...
     let tabId;
-    if (typeof tabOrTabId === 'number') {
+    if (Array.isArray(tabOrTabId)) {
+      tabOrTabId.map(this.updateLastAccessed.bind(this));
```
Comment (ssorallen) [inline app/js/tabmanager.js]: This case should immediately `return;`. This case knows tabId is a not a number, so it doesn't need to call `console.log` and report an error on line 241 below.

Comment (AsaAyers) [reply app/js/tabmanager.js]: It's fixed now.

Comment (ssorallen) [reply app/js/tabmanager.js]: Excellent, thanks again for this PR.

**Comment by ssorallen** [comment]
This throws an exception when it's called from `chrome.tabs.query` because `this` is not the correct context.

```
Error in response to tabs.query: TypeError: Cannot read property 'bind' of undefined
    at Object.updateLastAccessed [as callback] (chrome-extension://nejpokloncdgjkadecdpjooiladaecde/background.entry.js:843:45)
    at checkToClose (chrome-extension://nejpokloncdgjkadecdpjooiladaecde/background.entry.js:480:15)
```

**Comment by ssorallen** [comment]
I'll merge and fix in master.

**Pull request merged**